### DISCUSSION TITLE
p.250 section-085 リスト20.32 について

### DISCUSSION THREAD

**Discussion by yskgc** [budougumi0617/go_todo_app#31] [Ideas]
mux.go の実装について、
```golang
// 略
        l := &handler.Login{
		Service: &service.Login{
			DB:             db,
			Repo:           &r,
			JWTer: jwter, // -> TokenGenerator: jwter,
		},
		Validator: v,
	}
	mux.Post("/login", l.ServeHTTP)
// 略
```
上記の通り、`TokenGenerator: jwter,` が正しいかと思われます。
（[サンプルコード](https://github.com/budougumi0617/go_todo_app/blob/main/mux.go#L47) の方は `TokenGenerator: jwter,` となっています。）

**Comment by budougumi0617** [comment]
ご指摘ありがとうございました！正誤表に反映いたしました。
https://github.com/budougumi0617/go_todo_app/releases/tag/v1.0.3
